spec/services/idempotency/conflict_error_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Idempotency::ConflictError do
+  it { expect(described_class).to be < StandardError }
```

Comment:
Because this is returned to clients, how do you feel about adding a test for the message interpolating the key?

Reply:
yep, sure 👍